langchips-project/Program.cs

```diff
@@ -9,36 +9,83 @@ namespace langchips_project
 {
     internal class Program
     {
-        static void Main(string[] args)
+        static async Task Main(string[] args)
         {
             using (var context = new AppDbContext())
             {
-                context.Database.EnsureCreated();
+                try
+                {
+                    await context.Database.EnsureCreatedAsync();
 
+                    UserService userService = new UserService(context);
```

Comment:
https://henriquesd.medium.com/dependency-injection-and-service-lifetimes-in-net-core-ab9189349420

PS może UserService stworzyć w controllerze?